src/styled/feature-card.ts

```diff
@@ -0,0 +1,129 @@
+import styled from '@emotion/styled';
+import theme from '@mdb/flora/theme';
+import { PillCategory } from '../types/pill-category';
+import { HorizontalRule, Pill, Tag, TypographyScale } from '@mdb/flora';
+import Image from 'next/image';
+import { pillColorMap } from './pill-color';
+
+const FeaturedCardWrapper = styled('div')`
+    padding: 32px;
+    height: max-content;
+    border: 1px solid #e7eeec;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+    border-radius: ${theme.radii.inc50};
+
+    // Mobile
+    width: 375px;
+    // Desktop small
+    @media only screen and (min-width: ${theme.sizes.breakpoint.medium}) {
+        width: 696px;
+    }
+`;
+
+type PillProps = {
+    pillCategory: PillCategory;
+};
+
+const StyledPill = styled(Pill)`
+    margin-bottom: ${theme.space.elementXSmall};
+    background-color: ${(props: PillProps) => pillColorMap[props.pillCategory]};
+    // Desktop small
+    @media only screen and (min-width: ${theme.sizes.breakpoint.large}) {
+        padding: 8px 24px;
+        letter-spacing: 3px;
+    }
+`;
+
+const ThumbnailWrapper = styled('div')`
+    height: 96px;
+    width: 96px;
+    // Tablet
+    @media only screen and (min-width: ${theme.sizes.breakpoint.medium}) {
+        height: 180px;
+        width: 180px;
+    }
+`;
+
+const StyledThumbnail = styled(Image)`
+    border-radius: 8px;
+`;
+
+const CardHeader = styled('div')`
```

Comment:
If this is meant to act as a header I would use the `header` HTML tag. Also what do when I am not sure what tag to use I go to HTML5 doctor website 

http://html5doctor.com/element-index/#header

Reply:
`Represents the "header" of a document or section of a document.` I don't really think these cards are really functioning as a section of the document.